common/http_test.go

```diff
@@ -311,6 +311,18 @@ func TestUnmarshall(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestUnmarshallError(t *testing.T) {
+	v := struct {
+		Correct   string `json:"c"`
+		Incorrect int    `json:"i"`
+	}{}
+	ws := DatabricksClient{}
+	err := ws.unmarshall("/a/b/c", []byte(`{"c":"val", "i":"123"}`), &v)
+	require.Error(t, err)
+	require.EqualError(t, err,
+		`Invalid JSON received (22 bytes): {"c":"val", "i":"123"}, error: json: cannot unmarshal string into Go struct field .i of type int`)
```

Comment:
can you rather do it so that the error is `error: json: cannot unmarshal string into Go struct field .i of type int - '{"c":"val", "i":"12 (128 more bytes)'` (by using the `onlyNBytes` function)

Reply:
done